src/utils/utils.js

```diff
@@ -11,6 +11,8 @@ import platform             from 'platform'
 
 import * as constants   from '../constants/Constants'
 
+const language = navigator.language.split(/[-_]/)[0];  // language without region code
```

Comment:
@wildskyf let's have only one source of truth for language detection.

One suggestion is moving line 53 - 56 in src/index.js to src/utils/utils.js as an exported function.